healthcheck.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const router = express.Router({});
+
+router.get('/', async (_req, res, _next) => {
```

Comment:
nit: if `_next` is unused, it can be removed

```suggestion
router.get('/', async (_req, res) => {
```